src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -146,21 +159,55 @@ public NativeInterface(IConfiguration cfg)
      */
     AndroidJavaObject CreateNativeConfig(IConfiguration config) {
       var obj = new AndroidJavaObject("com.bugsnag.android.Configuration", config.ApiKey);
-      // the bugsnag-unity notifier will handle session tracking
-      obj.Call("setAutoCaptureSessions", false);
-      obj.Call("setEnableExceptionHandler", config.AutoNotify);
-      obj.Call("setDetectAnrs", config.AutoNotify && config.AutoDetectAnrs);
-      obj.Call("setCallPreviousSigquitHandler", false);
-      obj.Call("setDetectNdkCrashes", config.AutoNotify);
-      obj.Call("setEndpoint", config.Endpoint.ToString());
-      obj.Call("setSessionEndpoint", config.SessionEndpoint.ToString());
+      // configure automatic tracking of errors/sessions
```

Comment:
This comment is a bit misleading as automatic session tracking is disabled in the native notifier. Seems better to keep the original comment next to `setAutoTrackSessions`